api/turing/config/config.go

```diff
@@ -268,6 +268,8 @@ type RouterDefaults struct {
 	// {"experiment_engine_a": {"endpoint": "http://engine-a.com", "timeout": "500ms"},
 	//  "experiment_engine_b": {"endpoint": "http://engine-b.com", "timeout": "250ms"} }
 	Experiment map[string]interface{}
+
+	MonitoringURLFormat *string
```

Comment:
Is it possible to use the same variable for both jobs and routers?

Reply:
Yeah it's under a different struct.